public/Board.js

```diff
@@ -13,6 +15,8 @@ export default class Board {
     this.boardContainerElem.innerHTML = this.boardHtml();
     this.dealerCont = document.querySelector(".dealer");
     this.playersCont = document.querySelector(".players");
+    this.generatorCardObject = this.cardDeck.handsGenerator();
```

Comment:
Now that we have the `takeCard` method in the `Deck` entity we should use that instead of directly accessing the generator